application/classes/Ushahidi/Validator/Post/Create.php

```diff
@@ -227,6 +227,8 @@ public function checkTags(Validation $validation, $tags)
 
 	public function checkValues(Validation $validation, $attributes, $fullData)
 	{
+
+		$attributes = !empty($fullData['values']) ? $fullData['values'] : [];
```

Comment:
Actually PostUpdate usecase always pushing the full values array through anyway - so this change should make any difference.